modules/040-node-manager/webhooks/validating/instance_class

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
and better way rewrite hook on python and add tests
exampe hook
https://github.com/deckhouse/deckhouse/blob/main/modules/150-user-authn/webhooks/validating/group.py
https://github.com/deckhouse/deckhouse/blob/main/modules/150-user-authn/webhooks/validating/group_test.py